moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -62,7 +62,11 @@ class JogCalcs
 
   void stopMainLoop();
 
-  void haltOutgoingJogCmds();
+  /** \brief Stop publishing jog commands while keeping the main loop spinning and filters up to date */
+  void pauseOutgoingJogCmds();
```

Comment:
These shouldn't be in here, did you rebase onto the latest version of #1982?

Reply:
Sorry, I hadn't. Now up to date.